cli/kubectl_kadalu/storage_list.py

```diff
@@ -219,6 +222,10 @@ def fetch_status(storages, args):
             print()
         except FileNotFoundError:
             utils.kubectl_cmd_help(args.kubectl_cmd)
+        except ValueError:
+            print("Failed to get storage status since pv count"
```

Comment:
This is safe error to ignore. Not required to print on screen. Since all values are zero by default.

```
except ValueError:
    pass
```

Reply:
Add comment, when this can happen.

Reply:
Yes I thought of avoiding since 0 is set as default but one of the users in #359 had raised about the pending status but I retried by giving pvc more space than device but value error was not raised. I guess I'll just pass it as you're review.

Reply:
done!